CONTRIBUTING.md

```diff
@@ -1,21 +1,20 @@
 # Contributing to the Wazuh documentation
 
-In this document you'll find instructions about how to deploy a Python virtual environment that you can use to clone this repository, add a contribution and submit a pull request for review.
+This document provides instructions for deploying a Python virtual environment, cloning the repository, contributing changes, and submitting a pull request. It also explains how to report issues clearly so that we can resolve them quickly.
 
```

Comment:
```suggestion
This document explains how to set up a Python virtual environment, clone the repository, contribute changes, and submit a pull request. It also explains how to report issues clearly so that we can resolve them quickly.
```